includes/data/analytics-vendors-list.php

```diff
@@ -0,0 +1,473 @@
+<?php // phpcs:disable Squiz.Commenting.FileComment.Missing
```

Comment:
Can this file be placed at `includes/ecosystem-data/analytics-vendors.php`?

Reply:
Moved it to the `ecosystem-data` directory.